submissions/krylenger/dom-api-task/script.js

```diff
@@ -0,0 +1,144 @@
+    ],
+    avgPrice: "Average price: $10.000",
+  },
+  {
+    name: "OFFROAD",
+    img: "./images/offroad.png",
+    description: [
+      "The hallmarks of bikes specifically designed for off-road work tend to be very light weight, wire-spoked wheels, some form of protection under the engine in case of grounding, engine bars in case it drops, long travel suspension, hardly any screen in case you go over the top and usually \"knobbly\" tyres for improved grip.",
+      "They are popular for green laning on public byways / un-tarmacked roads.",
+    ],
+    models: ["HONDA CRF450RX", "HUSQVARNA FX450", "KTM 450XC-F"],
+    avgPrice: "Average price: $11.000",
+  },
+  {
+    name: "RETRO",
+    img: "./images/retro.png",
+    description: [
+      "Most of the retro models you can buy today started life as a sporty naked bike which has been modified to look more traditional, but without sacrificing all the modern tech and riding experience you would expect from a new bike.",
+      "They usually feature a single round headlight, simpler rounded clocks, stitched seats, more subtle colour schemes, rounded exhausts, loads of optional shiny bits and often wire spoked wheels or cast wheels designed to look a little bit like spokes from a distance.",
+      "Closely related are modern cafe racers which tend to start life as retro naked bikes but with the addition of low bars, a headlight fairing and pillion seat hump."
+    ],
+    models: ["Kawasaki Z900RS", "Triumph Speed Twin", "Moto Guzzi V7 III"],
+    avgPrice: "Average price: $10.000",
+  },
+];
+
+const navUl = document.querySelector('.nav__ul');
+const navLink = document.querySelectorAll('.nav__link');
+const mainHeader = document.querySelector('.main__header');
+const mainImage = document.querySelector('.main__img');
+const mainDescription = document.querySelector('.main__description');
+const mainDescriptionText = document.querySelector('.main__description-text');
+const mainUlBikes = document.querySelector('.main__ul-bikes');
+const mainAvgPrice = document.querySelector('.main__avg-price');
+
+const handleNavigationClick = (event) => {
+  const sectionName = event.target.textContent;
+  const section = database.find(item => item.name === sectionName);
+  mainHeader.textContent = section.name;
+  mainImage.setAttribute('src', section.img);
+  mainAvgPrice.textContent = section.avgPrice;
+
+  const fragmentBikes = document.createDocumentFragment();
+  const fragmentDescription = document.createDocumentFragment();
+  
+  section.models.forEach(model => {
+    let bike = document.createElement('li');
+    bike.textContent = model;
+    fragmentBikes.appendChild(bike);
+  })
+  section.description.forEach(description => {
+    let desc = document.createElement('p');
+    desc.textContent = description;
+    desc.setAttribute('class', 'main__description-p')
+    fragmentDescription.appendChild(desc);
+  })
+  navLink.forEach(link => {
+    link.style.color = 'lightgrey';
```

Comment:
Use rule CSS and [Element.classList](https://developer.mozilla.org/ru/docs/Web/API/Element/classList)

Reply:
✅ Done